extensions/elasticsearch/7.x/wazuh-template.json

```diff
@@ -215,7 +215,138 @@
       "data.ms-graph.threatDisplayName",
       "data.ms-graph.threatFamilyName",
       "data.ms-graph.title",
+      "data.ms-graph.@odata.type",
+      "data.ms-graph.activationLockBypassCode",
+      "data.ms-graph.activity",
+      "data.ms-graph.activityDateTime",

```

Comment:
We should only use `text` or `keyword` fields in the `index.query.default_field` property. Please remove `date` and `number` fields from this list.

Reply:
Done https://github.com/wazuh/wazuh/pull/25544/commits/d07998ceaf254bdc88e5ec7af991baed3f93c895